fdmi/plugins/fdmi_plugin_st.sh

```diff
@@ -55,18 +57,46 @@ do_some_kv_operations()
 			    -h ${lnet_nid}:$HA_EP -p $PROF_OPT \
 			    -f $M0T1FS_PROC_ID -s "
 
+		echo "Let's create an index and put {somekey:somevalue}"
 		"$M0_SRC_DIR/utils/m0kv" ${MOTR_PARAM}                                       \
 					index create "$DIX_FID"                            \
 					      put    "$DIX_FID" "somekey" "somevalue"      \
 					      get    "$DIX_FID" "somekey"                  \
-					      put    "$DIX_FID" "key1" "something1 anotherstring2 YETanotherstring3"      \
+				 || {
+			rc=$?
+			echo "m0kv failed"
+		}
+		if $interactive ; then echo "Press Enter to go ..." && read; fi
+
+		echo "Let put {key1: ***}"
+		"$M0_SRC_DIR/utils/m0kv" ${MOTR_PARAM}                                       \
+					index put    "$DIX_FID" "key1" "something1 anotherstring2 YETanotherstring3"      \
 					      get    "$DIX_FID" "key1"                     \
-					      put    "$DIX_FID" "key2" "something1_anotherstring2*YETanotherstring3"      \
+				 || {
+			rc=$?
+			echo "m0kv failed"
+		}
+		if $interactive ; then echo "Press Enter to go ..." && read; fi
+
+		echo "Let put {key2: ***}"
+		"$M0_SRC_DIR/utils/m0kv" ${MOTR_PARAM}                                       \
+					index put    "$DIX_FID" "key2" "something1_anotherstring2*YETanotherstring3"      \
```

Comment:
what this '*' denotes? is it a wildcard for multiple strings or something?
first two strings are joined with '_' and third with '*'. 
can we use consistent naming like
string1_string2_string3
or 
somestring1_somestring2_somestringN ???

Reply:
It does not mean anything.
It is literally the "*" char.
We can use any char to connect these three "substrings" that we are interested in.

Reply:
We don't have Wildcard or any regular expression at this moment.
We do literal binary string compare.